### PR TITLE
Add addContactPoint fluent method to CassandraClientOptions - fixes #3

### LINKED ISSUE #3: Add addContactPoint fluent method to CassandraClientOptions
Makes it easier to configure the client.

The contact points list could be empty by default. In this case, the client implementation would try on `localhost` only.

### PR DESCRIPTION
**Pull request by Sammers21**

### CODE REVIEW

**Comment by Sammers21** [comment]
@tsegismont , can you review?

File: src/main/java/io/vertx/cassandra/CassandraClientOptions.java [hunk 1]
```diff
@@ -55,6 +53,11 @@ public CassandraClientOptions setPort(int port) {
     return this;
   }
 
+  public CassandraClientOptions addContactPoint(String address) {
+    contactPoints().add(address);
+    return this;
+  }
+
   public List<String> contactPoints() {
     return contactPoints;
```
Comment (tsegismont) [inline src/main/java/io/vertx/cassandra/CassandraClientOptions.java]: Make sure to initialize `contactPoints` to an empty `ArrayList` if this method is invoked and the field is null

Comment (Sammers21) [reply src/main/java/io/vertx/cassandra/CassandraClientOptions.java]: Why we can't initialize `contactPoints` with an empty `ArrayList` when `CassandraClientOptions` is created?

Comment (Sammers21) [reply src/main/java/io/vertx/cassandra/CassandraClientOptions.java]: ah, i got it

Comment (Sammers21) [reply src/main/java/io/vertx/cassandra/CassandraClientOptions.java]: Done

**Comment by Sammers21** [comment]
@tsegismont , can you review one more time please?

**Comment by Sammers21** [comment]
@tsegismont , is it fine now?

**Comment by tsegismont** [comment]
@Sammers21 yep, LGTM. Thanks

**Pull request merged**